src/styles/themes/_default-theme.scss

```diff
@@ -470,7 +470,8 @@
   // ------ Stat tab theme ------
 
   // ------ Navbar theme ------
-  --navbar-background: #ffffffe6;
+  --navbar-background: var(--background-color);
```

Comment:
why are you dropping the opacity?

Reply:
Good catch, will add this back and move the hex to a new global variable.